src/style.css

```diff
@@ -1 +1,51 @@
-/* add styles here */
+.page {
+  --active-star: url(/src/images/star-active.svg);
+}
+
+.page__body {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-flow: column wrap;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 :first-child {
+  background-image: var(--active-star);
+}
+
+.stars--2 :nth-child(-n + 2) {
+  background-image: var(--active-star);
+}
+
+.stars--3 :nth-child(-n + 3) {
+  background-image: var(--active-star);
+}
+
+.stars--4 :nth-child(-n + 4) {
```

Comment:
The selector `.stars--4 :nth-child(-n + 4)` should not have a space before `:nth-child` if the intention is to target the first four `.stars__star` elements directly inside `.stars--4`. Use `.stars--4:nth-child(-n + 4)` instead.